### ISSUE TITLE
Tooltip dismissal in MainVC crashes

### ISSUE THREAD

**Issue by sofokli1** [toggl/mobileapp#5613]
## 💥 Crash Report
Dismissal of the some on-boarding tooltip in `ViewDidDissapear` produces a crash

### 👥 Impact
2.0.1 1k count 749 users
2.1 66 count 49 users (partial release)
2.1.1 41 count 36 users (partial release)
2.1.2 24 count 24 users (partial release)

### 🛰 Link
https://appcenter.ms/orgs/Toggl/apps/Toggl-iOS/crashes/errors/1288187084u/overview

### 📝  Exception & Stack Trace
`MainViewController.ViewDidDisappear (System.Boolean animated)
`
```cs
MainViewController.ViewDidDisappear (System.Boolean animated)
(wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain(int,string[],intptr,intptr)
UIApplication.Main (System.String[] args, System.IntPtr principal, System.IntPtr delegate)
UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName)
Application.Main (System.String[] args)
```

**Comment by sofokli1** [comment]
also in 2.1.3 https://appcenter.ms/orgs/Toggl/apps/Toggl-iOS/crashes/errors/537047696u/overview

**Comment by sofokli1** [comment]
I am not sure if this happen when app closes or when the user changes tab. What I do not like is that it is related one of the tooltips thus mainly new users are going to be affected by this and it makes for a bad experience for new users.

@amulware I would bump this in up in importance for the above reason (even just to investigate when it happens `when app closes or when the user changes tab`). If it turns out it happens when the app is closing i would't think much about it at that point.

**Comment by amulware** [comment]
Makes sense to me.

**Comment by juanlaube** [comment]
This looks like the crash you fixed today, @arturs105.

**Comment by Odrakir** [comment]
@juanlaube Can you direct me to that fix? Because I can't reproduce the crash.

**Comment by juanlaube** [comment]
@Odrakir it was fixed in ff3ec5c3, the crash is not happening anymore in 2.2.1.

**Comment by dshr** [comment]
> @Odrakir it was fixed in [ff3ec5c](https://github.com/toggl/mobileapp/commit/ff3ec5c3025782d41e0044c8f911261f61b9dfe5), the crash is not happening anymore in 2.2.1.

Closing this then, since this was closed by #5758
